Project3/index.html

```diff
@@ -0,0 +1,201 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Heebo:400,500,700,800,900|Mukta+Mahee:400,500,600,700,800"
+        rel="stylesheet">
+    <title>Timeline</title>
+</head>
+
+<body>
+    <h1>A day in Orlando</h1>
+    <figure id="hour1">
+        <h2>1. 6:30am</h2>
+        <a href="#hour1">
+            <img src="images/orlando1.jpg" alt="photo of two bedroom" width="300 " height="300">
```

Comment:
I think it's better to control the width and/or height of your `img` elements in CSS vs in your markup here:
```suggestion
            <img src="images/orlando1.jpg" alt="photo of two bedroom">
```

I'm only pointing this out on this one photo, but it's true for all of them.